features/integration/devmode.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Dev mode features
+  Background:
+    Given an algod v2 client
+
+  @devmode.offset
+  Scenario: Setting timestamp offsets in dev mode
+    When I set the timestamp offset to be 1234
+    And I get the timestamp offset
+    Then the timestamp offset should be 1234
```

Comment:
Is line 9 purely testing `GET /v2/devmode/blocks/offset/` ? If so, maybe it's also worth adding a step that calls `GET v2/block/{round}` and verifies that the offset had the desired effect? Might be tricky to make it non-flaky though...